tests/sync/util.js

```diff
@@ -72,6 +72,16 @@ const syncFromDisk = require('../../sync/syncFromDisk');
  * @property {"Set" | "Module"} [groupAssessmentsBy]
  */
 
+/**
```

Comment:
I turned on type checking for `tests/sync/assessmentsSync.test.js`, which necessitated updating a lot of things here.

Reply:
Nice!